solr_conf/schema.xml

```diff
@@ -631,6 +631,7 @@ v         currently supported on types that are sorted internally as strings
    <field name="sourceResource_type_id" type="string" indexed="true" stored="true" multiValued="true" />
    <field name="sourceResource_type_type" type="string" indexed="false" stored="false" multiValued="true" />
    <field name="sourceResource_type_providedLabel" type="string" indexed="true" stored="true" multiValued="true" />
+   <field name="sourceResource_type_name" type="string" indexed="true" stored="true" multiValued="true" />
```

Comment:
I'm not sure what the question is... I may be missing some important context.

I think it will be indexed if there is a schema entry for it, and not otherwise, right? https://github.com/dpla/KriKri/blob/develop/lib/krikri/search_index.rb#L266-L269